test/e2e/DCAHub/active-intervals.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { expect } from 'chai';
+import { BigNumber, utils } from 'ethers';
+import { ethers } from 'hardhat';
+import {
+  DCAHub,
+  DCAHub__factory,
+  DCAHubSwapCalleeMock,
+  DCAHubSwapCalleeMock__factory,
+  DCAPermissionsManager,
+  DCAPermissionsManager__factory,
+  IPriceOracle,
+} from '@typechained';
+import { TransactionResponse } from '@ethersproject/abstract-provider';
+import { constants, erc20, evm } from '@test-utils';
+import { contract } from '@test-utils/bdd';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { TokenContract } from '@test-utils/erc20';
+import { readArgFromEventOrFail } from '@test-utils/event-utils';
+import { buildSwapInput } from 'js-lib/swap-utils';
+import { SwapInterval } from 'js-lib/interval-utils';
+import { FakeContract, smock } from '@defi-wonderland/smock';
+import { snapshot } from '@test-utils/evm';
+
+contract('DCAHub', () => {
```

Comment:
Would you add a test where 
- Position created with swaps = 1 
- Swaps
=> No active swaps
- Someone deposits with swaps = 2 and rate less than the first position?
=> Interval gets active
- Swaps correctly

🙏 ?

Reply:
Added